src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMean.cpp

```diff
@@ -0,0 +1,274 @@
+
+Angle BrouwerLyddaneMean::getMeanAnomaly() const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("BrouwerLyddaneMean");
+    }
+
+    return anomaly_;
+}
+
+Angle BrouwerLyddaneMean::getTrueAnomaly() const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("BrouwerLyddaneMean");
+    }
+
+    return COE::TrueAnomalyFromMeanAnomaly(anomaly_, eccentricity_, 1e-15);
+}
+
+Angle BrouwerLyddaneMean::getEccentricAnomaly() const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("BrouwerLyddaneMean");
+    }
+
+    return COE::EccentricAnomalyFromMeanAnomaly(anomaly_, eccentricity_, 1e-15);
+}
+
+COE::CartesianState BrouwerLyddaneMean::getCartesianState(
+    const Derived &aGravitationalParameter, const Shared<const Frame> &aFrameSPtr
+) const
+{
+    const COE coe = this->toCOE();
+    return coe.getCartesianState(aGravitationalParameter, aFrameSPtr);
+}
+
+BrouwerLyddaneMean::BrouwerLyddaneMean(
+    const Length &aSemiMajorAxis,
+    const Real &anEccentricity,
+    const Angle &anInclination,
+    const Angle &aRaan,
+    const Angle &anAop,
+    const Angle &aMeanAnomaly
+)
+    : COE(aSemiMajorAxis, anEccentricity, anInclination, aRaan, anAop, aMeanAnomaly, COE::AnomalyType::Mean)
+{
+}
+
+Vector6d BrouwerLyddaneMean::Cartesian(
+    const COE::CartesianState &aCartesianState,
+    const Derived &aGravitationalParameter,
+    std::function<Vector6d(const Vector6d &)> toCOEVector
+)
+{
+    const COE coe = COE::Cartesian(aCartesianState, aGravitationalParameter);
+
+    bool possibleInaccuracyLongWritten = false;
+    bool inaccuracyCriticalAngleWritten = false;
+
+    if ((coe.getEccentricity() > 0.99) || (coe.getEccentricity() < 0.0))
+    {
+        throw ostk::core::error::RuntimeError(
+            String::Format("BrouwerLyddaneMean is applicable only if 0.0 < Ecc: [{}] < 0.99.", coe.getEccentricity())
+        );
+    }
+
+    const Real perigee = coe.getPeriapsisRadius().inMeters();
+    if (perigee < 3000000.0)
+    {
+        throw ostk::core::error::RuntimeError(
+            String::Format("BrouwerLyddaneMean is applicable only if Perigee radius: [{}] > 3000000.0 m.", perigee)
+        );
+    }
+    if (perigee < 6378000.0)
+    {
+        if (!possibleInaccuracyLongWritten)
+        {
+            std::cout << "Warning: For BrouwerLyddaneMean, when RadPer < 6378km, there is a "
+                         "possible inaccuracy due to singularity "
+                         "related with inside-of-earth orbit.\n";
+            possibleInaccuracyLongWritten = true;
+        }
+    }
+
+    const Real inclination_degrees = coe.getInclination().inDegrees();
+    if (inclination_degrees > 180.0)
+    {
+        throw ostk::core::error::RuntimeError(String::Format(
+            "BrouwerLyddaneMean is applicable only if inclination: [{}] < 180.0 degrees.", inclination_degrees
+        ));
```

Comment:
```suggestion
        throw ostk::core::error::RuntimeError(
            "BrouwerLyddaneMean is applicable only if inclination: [{}] < 180.0 degrees.", inclination_degrees
        );
```

nit: This should work. RuntimeError has built in string formatting